lib/code_map.js

```diff
@@ -40,6 +40,45 @@ var EventCode = {
   PREVIOUS_GTIDS_LOG_EVENT: 0x23
 };
 
+var CodeEvent = [
```

Comment:
Nice optimization, but now the `EventCode` object above is dead code, remove it!